### ISSUE TITLE
[BUG] [CART] Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by kstala** [shopware/frontends#1481]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

```
Element Location:
.bg-primary-500
<button class="text-white bg-primary-500 hover:bg-primary-600 justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md"> Add </button>
```

Element has insufficient color contrast of 3.67 (foreground color: #ffffff, background color: #3b82f6, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

### Environment

```markdown
- OS:
- Node:
- pnpm:
```
